src/screens/home-earn.tsx

```diff
@@ -37,7 +38,7 @@ export const HomeEarnScreen = () => {
   const [data, setData] = useState({
     ...initData,
     availableSum: visible.reduce(
-      (acc, w) => acc + app.getBalance(w.address),
+      (acc, w) => acc + formatBalanceToNumber(app.getBalance(w.address)),
```

Comment:
Here you can use `bn.add` and format sum at end component

Reply:
availableSum should not be a balance type. In this case we need to migrate all balance derivatives into Balance type